script.cinder/README.md

```diff
@@ -0,0 +1,175 @@
+###
+### Cinder - Instructions
+###
+
+Cinder is a free plug-in script for KODI that allows you to play random TV Episodes from a list of SMB shares.
+It is meant to be highly configurable and provide you a customized level of randomness that fits your taste.
+
+------------------------------------------------------------------------------------------------------------
+
+--= How to Configure Cinder =-- 
+
+-- Prerequisites --
+
+There are a few prerequisites that Cinder requires to function. 
+The following sections will provide a basic description of how you would setup Cinder to work on the 
+Windows 7 operating system. You can of course use any other operating system and can find specific details
+by searching the internet. Though the details will vary the concepts described below will be the same. 
+
+
+- Static IP Address -
+
+Cinder is designed to pull files from SMB shares that contain your media (more on that later). 
+The SMB shares need to be provided by a computer that resides on somewhere on your local network.
+We will call that system your media server for lack of a better term. Your media server can also run KODI.
+KODI instances running on other systems need to be able to find your media server over the network. 
+A common network configuration uses a service called DHCP which gives your computer a new IP address 
+every time it powers on. You want to avoid this since every time your media server restarts it will have
+a new address and the KODI clients running Cinder will not be able to find your media server.
+
+To avoid getting a new IP address each time your media server restarts it needs to be configured with a
+Static IP address that doesn't change. This is configured in:
+
+Control Panel\Network and Internet\Network Connections 
+
+Find your network adapter usually called "Local Area Connection" or "Wireless Network Connection"
+Double-click on your connection and select "Internet Protocol Version 4(TCP/IPv4)". 
+Next click the "Properties" button.
+In the pop-up click the "Use the following IP address" radio button and fill in your static IP address.
+
+The IP Address must be matching what is configured by your Router/WAP. For this example my WAP address is
```

Comment:
Out of curiosity, why are you using ip addresses and going through configuring SMB shares when it can be done at system level in Kodi by adding a video source? You would then use xbmcvfs to access the SMB shares.